src/SegmentedString.chpl

```diff
@@ -772,7 +774,7 @@ module SegmentedString {
       return (whereOffs, whereVals);
     }
 
-   proc segStrWhere(other: ?t, condition: [] bool, newLens: [] int) throws where t == owned SegString {
+    proc segStrWhere(other: ?t, condition: [] bool, newLens: [] int) throws where t == owned SegString {
```

Comment:
It looks like you might have an extra space here that may mess up alignment

Reply:
The original version is actually missing a space.